poprox-db/tests/test_create_scrape_table.py

```diff
@@ -0,0 +1,59 @@
+import psycopg2
+from sqlalchemy import create_engine
+
+# Database connection parameters
+conn_params = {
+    "dbname": "poprox",
+    "user": "postgres",
+    "password": "thisisapoproxpassword",
+    "host": "127.0.0.1",
+    "port": "5433",
+}
+
+# Create a SQLAlchemy engine
+engine = create_engine(
+    f'postgresql+psycopg2://{conn_params["user"]}:{conn_params["password"]}@{conn_params["host"]}:{conn_params["port"]}/{conn_params["dbname"]}'
+)
+
+
+# Function to test the creation of a new table and insertion of data
+def test_create_scrape_table():
+    # Connect to the PostgreSQL server
+    conn = psycopg2.connect(**conn_params)
+    cur = conn.cursor()
+
+    # Create a new table for testing
+    cur.execute("""
```

Comment:
This is not how we're managing create table commands we will need to clean this up.

To see how we're managing this type of change I recommend looking at the files in `poprox-storage/poprox-db/migrations/versions`

Reply:
Roger, I will clean up them later.

Reply:
Let me know if you want to pair up on that.